osh/hub/other/test_fixtures/users.json

```diff
@@ -0,0 +1,110 @@
+[
+{
+  "model": "kobo_auth.user",
+  "pk": 1,
+  "fields": {
+    "password": "pbkdf2_sha256$260000$Y1AhkH2Wye8AtwGVo4DkVh$siXy50B5w/8qJcKgquDm6ncKp0+S1pnX9Fy71Cog6oQ=",
+    "last_login": "2023-07-28T10:14:10.268",
+    "is_superuser": true,
+    "username": "admin",
+    "first_name": "",
+    "last_name": "",
+    "email": "kdudka@redhat.com",
```

Comment:
We should use a different email for the admin account.  @kdudka suggested to use `osh-admins@redhat.com` instead.

Reply:
does the mail list exist at all?

Reply:
Yes, it does.

Reply:
But using `admin@example.com` is fine as well. 😄